tools/python/boutiques/localExec.py

```diff
@@ -1177,3 +1169,18 @@ def loadJson(userInput, verbose=False):
         return json.loads(userInput)
     except ValueError:
         raise_error(ExecutorError, e)
+
+
+# Adds default values to input dictionary
+# for parameters whose values were not given
+def addDefaultValues(desc_dict, in_dict):
+    inputs = desc_dict['inputs']
+    for input in [s for s in inputs
```

Comment:
isn't `input` a reserved keyword?

Reply:
I copied this code over from another place so I didn't think about it, but I looked it up and apparently it's *not* a keyword, but will overwrite the built-in `input` function if used. We don't use `input` anywhere except the publisher, so it doesn't cause a problem, but it's bad practice. I'll change the name.